src/Database/Migrations/2020-12-28-223112_create_auth_tables.php

```diff
@@ -19,12 +19,13 @@ class CreateAuthTables extends Migration
 
     public function __construct(?Forge $forge = null)
     {
-        parent::__construct($forge);
-
         /** @var Auth $authConfig */
         $authConfig       = config('Auth');
         $this->tables     = $authConfig->tables;
         $this->attributes = ($this->db->getPlatform() === 'MySQLi') ? ['ENGINE' => 'InnoDB'] : [];
+
+        $this->DBGroup = $authConfig->DBGroup;
+        parent::__construct($forge);
```

Comment:
for fix phpunit test:
```suggestion
        parent::__construct($forge);

        /** @var Auth $authConfig */
        $authConfig = config('Auth');

        if ($authConfig->DBGroup !== null) {
            $this->DBGroup = $authConfig->DBGroup;
        }

        $this->tables     = $authConfig->tables;
        $this->attributes = ($this->db->getPlatform() === 'MySQLi') ? ['ENGINE' => 'InnoDB'] : [];
```